dove/src/docs/options.rs

```diff
@@ -0,0 +1,128 @@
+// Copyright (c) The Diem Core Contributors
+// SPDX-License-Identifier: Apache-2.0
+use serde::{Deserialize, Serialize};
+
+/// Docgen options.
+#[derive(Deserialize, Serialize, Debug, Clone, PartialEq, Eq)]
+pub struct DocgenOptions {
```

Comment:
It is not clear how it's different from original one. Could you clarify (here, in conversation)?

Reply:
The original structure has an output_directory field and a doc_path . These fields are calculated and should not be available to the user for modification.